ksql-engine/src/test/java/io/confluent/ksql/util/SerDeUtilTest.java

```diff
@@ -117,4 +119,78 @@ public void shouldReturnCorrectKsqlSchema() {
                                                                                        .STRING_SCHEMA));
   }
 
+
+  @Test
+  public void shouldFailForUnsupportedUnion() {
+
+    String avroSchemaStr = "{\n"
+                           + "  \"type\": \"record\",\n"
+                           + "  \"name\": \"LOGON\",\n"
+                           + "  \"namespace\": \"ORCL.SOE2\",\n"
+                           + "  \"fields\": [\n"
+                           + "    {\n"
+                           + "      \"name\": \"table\",\n"
+                           + "      \"type\": [\n"
+                           + "        \"string\"\n"
+                           + "      ],\n"
+                           + "      \"default\": null\n"
+                           + "    },\n"
+                           + "    {\n"
+                           + "      \"name\": \"LOGON_DATE\",\n"
+                           + "      \"type\": [\n"
+                           + "        \"null\",\n"
+                           + "        \"int\",\n"
+                           + "        \"long\"\n"
+                           + "      ],\n"
+                           + "      \"default\": null\n"
+                           + "    }\n"
+                           + "  ],\n"
+                           + "  \"connect.name\": \"ORCL.SOE2.LOGON\"\n"
+                           + "}";
+
+    try {
+      Schema schema = SerDeUtil.getSchemaFromAvro(avroSchemaStr);
+      fail();
+    } catch (KsqlException ksqlException) {
+      assertThat("", ksqlException.getMessage(), equalTo("Union type cannot have more than two "
```

Comment:
if you aren't going to provide a message in the `assertThat(..)` you can omit the first string param, i.e., just use `assertThat(ksqlException.getMessage(), equalTo("...."))`